spec/lrama/states_spec.rb

```diff
@@ -2241,21 +2253,42 @@ class    go to state 5
 
             $default  reduce using rule 6 (rel_expr)
 
-            relop  go to state 11
+            relop  go to state 12
```

Comment:
I think this change is not intentional in other words states 11, 12, 13 and 14 should not be crated. What do you think about it?